doc/ref_model/chapters/chapter02.md

```diff
@@ -266,17 +266,45 @@ By trying to sort workloads into different categories based on the requirements
 <a name="2.4"></a>
 ## 2.4 Cloud Infrastructure Profiles
 
-Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
+Infrastructure profiles are used to tag infrastructure (such as hypervisor hosts, or Kubernetes worker nodes) and associate it with a set of capabilities exploitable by the workloads. *Workload profiles* are requirements expressed as workload metadata, indicating what kind of infrastructure they must run on to achieve functionality and/or the intended level of performance.
+Two profile *layers* are proposed:
+
+- The top level **profiles** represent macro-characteristics that partition infrastructure into separate pools, ie: an infrastructure object can belong to one and only one profile, and workloads can only be created using a single profile. Workloads requesting a given profile **must** be instantiated on infrastructure of that same profile.
+- For a given profile, **flavours** represent small deviations from (or further qualification, such as sizing) the profile that do not require partitioning the infrastructure into separate pools, but that have specifications with a finer granularity of the profile. Flavours can be *optionally* requested by workloads that want a more granular control over what infrastructure they can run on, ie: an infrastructure resource can have **more than one flavour** attached to it, and workloads can request VMs to be instantiated on infrastructure with a certain flavour. Workloads requesting a given flavour **must** be instantiated on infrastructure with that same flavour. It is allowed to instantiate workloads on infrastructure with more flavours than what is requested, as long as the minimum requirements are satisfied.
```

Comment:
In RM Chapter4, compute flavour represents the resources needed by a workload (vCPU, RAM...).
Profile flavours can represent different options within an infrastructure profile, but not sizing as written in "(or further qualification, such as sizing)"

Reply:
yes, we can choose to either keep the **_infrastructure flavours_** as 

- qualitative-only boolean labels (eg. has-smartnic, or even high/low memory, as a general indication without specifying the amount of ram), 
- or flesh them out as quantitative (eg. higmem.512 is a label for a node that has exactly 512GB of RAM)

i'd prefer the first one personally, but suspect we'll be forced to be more prescriptive by workloads wanting to know the exact infra specs from the flavour labels

Reply:
will rename `flavour` to `infrastructure flavour`

Reply:
ref. https://github.com/cntt-n/CNTT/issues/2334